internal/entity/article.go

```diff
@@ -0,0 +1,122 @@
+package entity
+
+import (
+	"encoding/xml"
+)
+
+type (
+	ArticleListExternal struct {
```

Comment:
XML types are supposed to be defined on XML client level, no? otherwise you mix types from different layers

Reply:
fixed